Language/Functions/Interrupts/noInterrupts.adoc

```diff
@@ -61,6 +61,14 @@ void loop() {
   // other code here
 }
 ----
+[%hardbreaks]
+
+
+[float]
+=== Notes and Warnings
+Note that disabling interrupts on the Arduino Nano 3.3v BLE will make the board
```

Comment:
```suggestion
Note that disabling interrupts on the Arduino boards with native USB capabilities (e.g., link:https://docs.arduino.cc/hardware/leonardo[Leonardo^]) will make the board
```

The information should be made generally applicable, rather than arbitrarily specific to a single board.

I chose Leonardo as the example of the affected class of boards because I think it is the most well known.

Reply:
Okay, I only had the Nano 3.3v BLE to test so wasn't sure if it was actually generally applicable. But am happy to make the change 👍 . Is there a reason to link the leonardo specifically?